extensions/wikia/EditPageLayout/js/plugins/Autoresizer.js

```diff
@@ -107,7 +109,7 @@
 			return dimensions;
 		},
 
-		resize: function() {
+		resize: function() {	
 			switch(this.mode) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "default" clause to this "switch" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)

[project-id]: wikia:app
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement with "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1301)

[project-id]: wikia:app